x-pack/platform/plugins/shared/task_manager/server/monitoring/task_run_calculators.ts

```diff
@@ -64,3 +64,18 @@ export function createMapOfRunningAveragedStats<T>(runningAverageWindowSize: num
     return asRecordOfValues();
   };
 }
+
+// Use interquartile range (IQR) to determine outliers
+export function filterOutliers(values: number[]): number[] {
+  if (values.length < 4) {
+    return values;
+  }
+
+  const p25 = stats.percentile(values, 0.25);
```

Comment:
Do we really want to filter the quick/small ones?  That seems like it will drive the duration values "up" a bit, in aggs, I would think.  Seems like we just want to remove the large outliers.

Also wondering about the general calculation here.  Where did the `1.5` come from, below?  It's making me think! :-). And google.  Found this: https://en.wikipedia.org/wiki/Interquartile_range#Outliers

Reply:
Yeah that makes sense! I can remove the lower range check
Yes that's my bad, I should have included more information on IQR! I did some research to figure out the best way to filter outliers and IQR and using the st deviation were the main options

Reply:
hehe, ya I poked around a little, looked at stdev and then ran into the 1.5 factor somewhere :-)

I was thinking stdev might work better, but after reading, seemed like percentile probably makes the most sense.

Reply:
Removed in this commit, https://github.com/elastic/kibana/pull/229022/commits/2041bffef133558be523214c6157f5afb96c06e7